packages/fxa-react/components/Checkbox/index.stories.tsx

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Per our Slack chat, I disagree and think this is generic enough that it could belong in fxa-react for future re-use. But I also do not mind it in fxa-settings. Let's stick it there for now.